docs/contrib/DeveloperGuide.md

```diff
@@ -2,7 +2,7 @@
 
 ## Building
 
-Instructions for building are available [here](../user/GettingStarted.md).
+Instructions for building are available [here](getting:started:setup:build).
```

Comment:
will be changed in [this PR](https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/pull/771)

Reply:
there isn't a way that can work on both, unfortunately. Maybe if there's a parser other than myst that can allow for cross-referencing to work on both